scripts/apps/authoring/views/article-edit.html

```diff
@@ -526,7 +527,8 @@
      data-required="schema[field._id].required">
     <label translate>{{ field.display_name }}</label>
     <div sd-item-association
-         data-allow-video="field.field_type === 'video'"
+         data-allow-video="field.field_type === 'video' || field.field_type === 'media'"
```

Comment:
I thought we would use filed type media also for picture and video, so those would not be needed. and there would be something like `field.field_options.allowed_types` with eg `{picture: 1, video: 1, audio: 0}` or just `{picture: 1}`